kano_updater/utils.py

```diff
@@ -135,21 +128,37 @@ def run_for_every_user(cmd):
         run_cmd_log("sudo su -c '{cmd}' - {user}".format(cmd=cmd, user=user))
 
 
-def is_server_available():
-    install_ping()
-    import ping
-
-    lost_packet_percent = ping.quiet_ping(REPO_SERVER, timeout=3, count=5)[0]
+def is_server_available(REPO_SERVER, count = 5):
```

Comment:
I would honestly suggest not reimplementing this function with `subprocess` as opposed to using `ping`. If it can be done through the module I think that'll be more maintainable and safer. In that case you'd also have to make the `kano-updater` package depend on `python-ping` in the control file.

Reply:
+1